docs/how_to_guides/msm_estimation_exercise.ipynb

```diff
@@ -0,0 +1,1295 @@
+{
```

Comment:
<p>Although it seems handy, it disguises what is going on which is not suitable for the documentation. Please be explicit even if it leads to repititions.</p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/OpenSourceEconomics/respy/pull/356/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='9'/>

Reply:
True, I agree. I was wondering whether I should change it. It definitely might be more confusing than helpful to readers.